src/utils/nimbledroid/index.jsx

```diff
@@ -120,8 +120,10 @@ export const generateSitesTableContent = (
     yellow: 0,
     green: 0,
   };
+
   const tableHeader = packageIds.map(packageId => CONFIG.packageIdLabels[packageId]);
-  tableHeader.push('% from target');
+  tableHeader.push(`% from ${CONFIG.packageIdLabels[CONFIG.baseProduct]}`);
+
```

Comment:
Rather than typing `CONFIG.packageIdLabels` constantly you could do `const { baseProduct, packageIdLabels} = CONFIG;`